frame/staking/src/pallet/mod.rs

```diff
@@ -1237,7 +1233,7 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// (Re-)set the controller of a stash.
+		/// (Re-)sets the controller of a stash to the stash itself.
```

Comment:
I would add some explanation about why this function has the signature it has, namely the history and son. Seeing this for the first time without the context might be confusing